### PR TITLE
Update bash_completion docs

### PR DESCRIPTION
**Pull request by dimaburmistrov**
Problem:
- "have" function is deprecated (and not shipped)
- "_argcomplete" in similar situation
- all configuration is outside of patool

Solution:
- remove useless completion
- add instructions to README

### CODE REVIEW

**Comment by dimaburmistrov** [comment]
Alternative bash completion written in Bash: https://gist.github.com/dimaburmistrov/23f337cf103efb77cc28f2774f31cef6

It must be synchronized manually but it doesn't require extra packages.

It can be optionaly mentioned in README - if you want to. Otherwise - link is JFYI.

File: doc/README.txt [hunk 1]
```diff
@@ -72,6 +72,17 @@ patoolib.search_archive("def urlopen", "python3.3.tar.gz")
...
+- run `activate-global-python-argcomplete` to create `/etc/bash_completion.d/python-argcomplete.sh`,
+- add `eval "$(register-python-argcomplete patool)"` to your `~/.bashrc`.
+
+Open new bash session.
+
+More information about `argcomplete` installation and usage can be found [here](https://github.com/kislyuk/argcomplete).
+
 Test suite status
 ------------------
 Patool has extensive unit tests to ensure the code quality.
```
Comment (yarikoptic) [inline doc/README.txt]: and what needs to be done to the provided `patool.bash-completion` ?

File: doc/README.txt [hunk 2]
```diff
@@ -72,6 +72,17 @@ patoolib.search_archive("def urlopen", "python3.3.tar.gz")
...
+Bash completion
+----------------
+To enable bash completion
+- install python `argcomplete` package,
+- run `activate-global-python-argcomplete` to create `/etc/bash_completion.d/python-argcomplete.sh`,
+- add `eval "$(register-python-argcomplete patool)"` to your `~/.bashrc`.
+
+Open new bash session.
+
+More information about `argcomplete` installation and usage can be found [here](https://github.com/kislyuk/argcomplete).
```
Comment (yarikoptic) [inline doc/README.txt:84]: FWIW in datalad (and other projects) we just added a command such as 
```
$> datalad shell-completion --help-np
Usage: datalad shell-completion [-h] [--version]

Display shell script for enabling shell completion for DataLad.

Output of this command should be "sourced" by the bash or zsh to enable
shell completions provided by argcomplete.

Example:

    $ source <(datalad shell-completion)
    $ datalad --<PRESS TAB to display available option>

optional arguments:
  -h, --help, --help-np
                        show this help message. --help-np forcefully disables the use of a pager for displaying the help
                        message
  --version             show the module and its version which provides the command
```
so users (and myself) doesn't need to RTFM and can find/spot such command among available in the tool CLI

Comment (dimaburmistrov) [reply doc/README.txt:84]: It's was long time ago - can't catch up detail now. I'd like to suggest closing this request as outdated.

**Comment by wummel** [comment]
Autocompletion uses the argcomplete python module, and is documented in README.md. So this pull request is outdated, closing. Thanks anyway for the patch.

**Pull request closed without merging**